README.md

```diff
@@ -76,6 +76,9 @@ The name of the assert pretty much tells you everything:
 * `assert_le!(a, b)` -> Will panic if a is not greater than or equal to b
 * `assert_range!((x..y), b)` -> Will panic if b is not within the range [x, y)
 * `assert_nrange!((x..y), b)` -> Will panic if b is within the range [x, y)
+* `assert_true!(a)` -> Will panic if a is not true
+* `assert_false!(b)` -> Will panic if a is not false
```

Comment:
These aren't required (refer to previous comments)

Reply:
assert_true!() and assert_false!() are intended to provide a clear statement of test intention.
assert!() is a broad low-level function and works at a different level of abstraction than the testing assertions (assert_eq!(), etc...). The bare assert is a foundation function. Xunit-based test frameworks reserve the bare assert for checks of the test integrity (typically in the "assemble" and "act" stages of assemble, act, assert), and not for evaluating the system under test. Functions derived from assert!() with clearer names are

This approach is most likely biased by my previous experience with C++ and Google Test. Google Test strongly separates the ideas of the tests of the system under test, from general purpose checks of the test program's integrity. It may or may not be the right strategy for this library.

Reply:
P.S. Thank you for the feedback.